README.md

```diff
@@ -1555,7 +1555,23 @@ V8 is Google’s open source high-performance JavaScript and WebAssembly engine,
 
 ## Q. ***How V8 compiles JavaScript code?***
 
-*ToDo*
```

Comment:
This answer can be more optimized. Please refere this blog for more information

https://cabulous.medium.com/how-v8-javascript-engine-works-5393832d80a7

Reply:
Sure, I'll describe it in more detail.

Reply:
@learning-zone, 
I've been studying/writing it for a while, and I've come to this. Do you think I can detail it better in any part?